lint.list

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Please note this temporary file is removed in https://github.com/OpenBazaar/openbazaar-desktop/pull/229/commits/14cabeaba589f34b57f1455637545eb95008b83b